component/src/main/java/org/wso2/extension/siddhi/io/ibmmq/source/IBMMessageConsumer.java

```diff
@@ -0,0 +1,113 @@
+import org.wso2.siddhi.core.stream.input.source.SourceEventListener;
+
+import java.nio.ByteBuffer;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import javax.jms.JMSException;
+import javax.jms.MapMessage;
+import javax.jms.Message;
+import javax.jms.MessageConsumer;
+import javax.jms.TextMessage;
+
+/**
+ * IBM MQ message consumer class which retrieve messages from the queue.
+ **/
+public class IBMMessageConsumer implements Runnable {
+    private static final Logger LOGGER = LoggerFactory.getLogger(IBMMessageConsumer.class);
+    private SourceEventListener sourceEventListener;
+    private MQConnection connection;
+    private MessageConsumer messageConsumer;
+    private Boolean isPaused = false;
+    private ScheduledFuture scheduledFuture;
+
+
+    public IBMMessageConsumer(SourceEventListener sourceEventListener, MQConnection connection,
+                              MessageConsumer messageConsumer) throws JMSException {
+        this.sourceEventListener = sourceEventListener;
+        this.connection = connection;
+        this.messageConsumer = messageConsumer;
+        this.connection.start();
+    }
+
+    @Override
+    public void run() {
+        while (true) {
+            try {
+                if (!isPaused) {
+                    Message message = messageConsumer.receive();
+                    if (message instanceof MapMessage) {
+                        Map<String, Object> event = new HashMap<>();
+                        MapMessage mapEvent = (MapMessage) message;
+                        Enumeration<String> mapNames = mapEvent.getMapNames();
+                        while (mapNames.hasMoreElements()) {
+                            String key = mapNames.nextElement();
+                            event.put(key, mapEvent.getObject(key));
+                        }
+                        sourceEventListener.onEvent(event, null);
+                    } else if (message instanceof TextMessage) {
+                        String event = ((TextMessage) message).getText();
+                        sourceEventListener.onEvent(event, null);
+                    } else if (message instanceof ByteBuffer) {
+                        sourceEventListener.onEvent(message, null);
+                    }
+                } else {
+                    scheduledFuture.wait();
+                    return;
+                }
+            } catch (JMSException e) {
+                throw new IBMMQSourceAdaptorRuntimeException("Exception occurred during consuming messages " +
+                        "from the queue: " + e.getMessage(), e);
+            } catch (InterruptedException e) {
+                LOGGER.error("Error while pausing the consuming messages: "
+                        + e.getMessage(), e);
```

Comment:
Please get these two lines into a single line

Reply:
This class is removed by 6e40355446fd7c544b8b0332258d5ca6ba556a0f